run.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this file here? Looks the same as `scripts/run.sh`.

Reply:
No, we do not need this one. Thanks.